### PR TITLE
Fix GH#19371: Set text line's text color

### PR DESCRIPTION
**Pull request by Jojo-Schmitz**
Resolves: #19371

IMHO better than #19391, as it also changes the texts on a change of the line color directly and not only after save/close/reopen

### CODE REVIEW

**Comment by Jojo-Schmitz** [comment]
Actually apparently Mu4 needs both

**Review by oktophonie — APPROVED** [review]

**Review by cbjeukendrup — APPROVED** [review]

File: src/engraving/rendering/stable/tlayout.cpp
Comment (cbjeukendrup) [inline src/engraving/rendering/stable/tlayout.cpp:1]: @Jojo-Schmitz The changes need to be made in the `…/dev/tlayout.cpp`, not `…/stable/tlayout`. As far as I know, `stable` is not used anywhere, so after moving the change to `dev` it needs to be retested (and also rechecked whether the changes in textlinebase.cpp are still necessary).

Comment (Jojo-Schmitz) [reply src/engraving/rendering/stable/tlayout.cpp:1]: Oops.

Comment (Jojo-Schmitz) [reply src/engraving/rendering/stable/tlayout.cpp:1]: Seems it is used in my build though, else it'd not have any effect locally?
And apparently without the changes in tlayout.cpp reading the colors didn't work locally either.

Anyway, added the changes to the other tlayout.cpp, and added another commit disabling the chnages in textline.cpp, let's see what the tests say, esp. the vtests

Comment (Jojo-Schmitz) [reply src/engraving/rendering/stable/tlayout.cpp:1]: Apparently indeed the changes in textline.cpp don't seem needed

Comment (Jojo-Schmitz) [reply src/engraving/rendering/stable/tlayout.cpp:1]: I'll leave the change in dev and stable, better save than sorry ;-)

**Review by cbjeukendrup — APPROVED** [review]

**Pull request merged**